src/models/furiousLegend.js

```diff
@@ -63,10 +63,24 @@ nv.models.furiousLegend = function() {
 
                 seriesShape = series.select('rect');
 
-                seriesEnter.append('g')
+//                 seriesEnter.append('g')
```

Comment:
@gowrishankaran why not just delete the old code? We have git for version control if we ever need to get it back